airbyte-container-orchestrator/Dockerfile

```diff
@@ -4,8 +4,8 @@ FROM ${JDK_IMAGE} AS orchestrator
 
 ARG DOCKER_BUILD_ARCH=amd64
 
-RUN amazon-linux-extras install -y docker
-RUN yum install -y jq tar && yum clean all
+RUN dnf install -y docker
```

Comment:
Don't include the change to move to `dnf` unless the base container actually moves to AL2023. The `dnf` command is not available in Amazon Linux 2.

Reply:
Build fails without this change